src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import './mixins';
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  height: 100vh;
+  width: 100vw;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    bottom: 50vmin;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    z-index: 1;
+
+    @include rotate-animation(3600s, steps(60));
```

Comment:
Ensure that the `rotate-animation` mixin is defined in the `mixins.scss` file and that it takes the correct parameters. If it's not defined, the SCSS will not compile correctly.